gen3/bin/kube-setup-secrets.sh

```diff
@@ -132,15 +132,6 @@ if gen3_time_since secrets_sync is 120; then
   gen3 secrets sync || true
 fi
 
-# mariner
-cd "$(gen3_secrets_folder)"
-if ! g3kubectl get secret workflow-bot-g3auto > /dev/null 2>&1; then
-  credsFile=$(mktemp -p "$XDG_RUNTIME_DIR" "creds.json_XXXXXX")
-  jq -r .mariner < creds.json > "$credsFile"
-  g3kubectl create secret generic workflow-bot-g3auto "--from-file=credentials.json=${credsFile}"
-  rm "$credsFile"
-fi
-
```

Comment:
Is this secret no longer needed? I couldn't find where this creation had been moved to :( Maybe I am being bling

Reply:
yeah - it's in kube-setup-mariner now - was moved a while ago, but forgot to clean this up: https://github.com/uc-cdis/cloud-automation/blob/master/gen3/bin/kube-setup-mariner.sh#L10
THanks for the quick review!